src/commonMain/kotlin/indexer/models/chain/OnChainDelegationResponse.kt

```diff
@@ -10,10 +10,15 @@ data class OnChainDelegationResponse(
 
 @Serializable
 data class OnChainDelegationObject(
+    val delegation: OnChainDelegationInfo? = null,
```

Comment:
Hmmm. Is the change coming from codegen?

Reply:
no I didn't codegen, this is just what the object is